src/app/Page.js

```diff
@@ -11,15 +13,47 @@ export default class Page extends React.Component<{|+children: Node|}> {
     return (
       <div>
         <header>
-          <nav>
-            <ul style={{listStyle: "none", paddingLeft: 0, margin: 0}}>
+          <nav className={css(style.nav)}>
+            <ul className={css(style.navList)}>
+              <li className={css(style.navItem, style.navItemLeft)}>
+                <Link to="/" className={css(style.navLinkTitle, style.navLink)}>
+                  SourceCred
+                </Link>
+              </li>
               {routeData.map(({navTitle, path}) =>
                 NullUtil.map(navTitle, (navTitle) => (
-                  <li key={path} style={{display: "inline", marginRight: 10}}>
-                    <Link to={path}>{navTitle}</Link>
+                  <li
+                    key={path}
+                    className={css(style.navItem, style.navItemRight)}
+                  >
+                    <Link to={path} className={css(style.navLink)}>
+                      {navTitle}
+                    </Link>
                   </li>
                 ))
               )}
+              <li className={css(style.navItem, style.navItemRight)}>
+                <a
+                  className={css(style.navLink, style.navLinkHover)}
```

Comment:
This `navLinkHover` now evaluates to `undefined` and can be removed.
Same below.